src/shared/api/indexer/connector.tsx

```diff
@@ -1,11 +1,11 @@
-// @ts-nocheck
-/* eslint-disable -- disabling this file as this was created before our strict rules */
```

Comment:
The lack of types here was pretty brutal as it wasn't detecting the import changes needed. Went ahead and added types.